db/postgres/migrations/1700581441938-FixForRefreshTokenTable.ts

```diff
@@ -0,0 +1,38 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class FixForRefreshTokenTable1700581441938 implements MigrationInterface {
```

Comment:
What does it fixes?

Reply:
It was not allowed to create more than one token. I changed it from a one-to-one relationship to a one-to-many relationship.